LibrarySystemService/src/main/java/com/target/ready/library/system/service/LibrarySystemService/Controller/LibrarySystemController.java

```diff
@@ -53,5 +54,10 @@ public Book findByBookId(@PathVariable int bookId){
     public List<Book> findBookByCategoryName(@PathVariable String categoryName){
         return libraryService.findBookByCategoryName(categoryName);
     }
+    @PutMapping("bookUpdaterService/{id}")
```

Comment:
Api names should be noun and in snake case.

Reply:
Sure